### PR TITLE
Feature/2069 registration list

### PR DESCRIPTION
**Pull request by ArronChaveAC**
https://github.com/Scottish-Natural-Heritage/Licensing/issues/2069

### CODE REVIEW

File: src/controllers/renewal-login.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import jwt from 'jsonwebtoken';
 import jwkToPem from 'jwk-to-pem';
 import config from '../config.js';
 import axios from '../http-request.js';
-import {ReturnState} from './_base.js';
+import {monthsFromNow, yearsAgo} from '../utils/date-utils.js';
 
 /**
  * Get the TR-API's public key.
```
Comment (RAnderson96) [inline src/controllers/renewal-login.js:8]: Comments need to be updated in line with your code

File: src/http-request.js [hunk 2]
```diff
@@ -77,10 +105,15 @@ const mockAxios = {
    * @returns {Promise<any>} A fake response.
    */
   async get(url) {
+    console.log('🚀 ~ get ~ url:', url);
```
Comment (RAnderson96) [inline src/http-request.js:108]: This console.log probably needs removed

**Review by RAnderson96 — CHANGES_REQUESTED** [review]

File: src/controllers/renewal-login.js (hunk 1 above)
Comment (RAnderson96) [reply src/controllers/renewal-login.js:8]: lines 23 - 42 of src/controllers/renewal-login.js

Comment (ArronChaveAC) [reply src/controllers/renewal-login.js:8]: I'm more in favour of removing them than deleting them, the code is self explanatory and comments add a useless overhead

File: src/http-request.js (hunk 2 above)
Comment (RAnderson96) [reply src/http-request.js:108]: I think you missed this

**Review by RAnderson96 — CHANGES_REQUESTED** [review]

**Review by RAnderson96 — APPROVED** [review]

**Pull request merged**